client/Lykke.Service.Assets.Client/Lykke.Service.Assets.Client.csproj

```diff
@@ -25,8 +25,17 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Lykke.Cqrs" Version="6.0.0" />
     <PackageReference Include="Lykke.Logs" Version="6.0.0" />
     <PackageReference Include="Microsoft.Rest.ClientRuntime" Version="2.3.13" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="ReadModels\" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\contract\Lykke.Service.Assets.Contract\Lykke.Service.Assets.Contract.csproj" />
```

Comment:
Contract will be delivered via separate nuget? If you want to include this assembly to the client nuget, then extra csproj configuration is needed. See https://stackoverflow.com/questions/46376256/visual-studio-2017-nuget-pack-exclude-project-reference-as-another-nuget-referen and better don't forget to include xlm file too

Reply:
This project is already delivered via separate nuget, but I think there should be only one package. Thanks for the consideration, I'll fix it.